docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md

```diff
@@ -112,23 +112,6 @@ The following template variables are available when expanding annotations and la
 | $values | The values of all reduce and math expressions that were evaluated for this alert rule. For example, `{{ $values.A }}`, `{{ $values.A.Labels }}` and `{{ $values.A.Value }}` where `A` is the `refID` of the expression. This is unavailable when the rule uses a classic condition. |
 | $value  | The value string of the alert instance. For example, `[ var='A' labels={instance=foo} value=10 ]`.                                                                                                                                                                                  |
 
-#### Template functions
```

Comment:
Missing section.

Reply:
This is a revert?

Reply:
The table under "Template functions" section was added for the PR and it is relevant information? If yes, we need to add that back.